CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - Fix: bug having entities without attibutes AND matching registrations made the attr-less entity not be returned by GET /v2/entities (#3176)
 - Fix: bug when using "legacy" format for custom v2 notifications (#3154)
 - Fix: check for providers is done only in UPDATE or REPLACE actionType case in update entity logic (#2874)
+- Add: transient entities functionality (new virtual attribute: dateExpired) (#3000)
```

Comment:
Suggested change:

```
- Add: transient entities functionality (based on "dateExpired" attribute) (#3000)
```

The different may be subtle... but I understand it is more precise, as at the end dateExpired is similar to location, but not actually virtual attribute (in the sense e.g. dateModified or dateCreated are).

Reply:
Right, that description was there before we decided not to use dateExpires as a virtual attribute.

Reply:
Fixed in ae70f62